samples/bluetooth/unicast_audio_client/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_ADV_PERIODIC=y
+
+# For LC3 the following configs are needed
+CONFIG_LIBLC3CODEC=y
+CONFIG_FPU=y
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+CONFIG_NEWLIB_LIBC=y
```

Comment:
What do we need from NEWLIB_LIBC? `math.h`?

Reply:
Yes for the float math functions